odoo/models.py

```diff
@@ -3002,6 +3002,10 @@ def _update_field_translations(self, field_name, translations, digest=None):
         """
         self.ensure_one()
 
+        installed_lang = list(code for code, _ in self.env['res.lang'].get_installed())
```

Comment:
Maybe use directly a set comp, and do a join instead of hardcoding the repr of the set ?
```
installed_lang = {code for code, _ in self.env['res.lang'].get_installed()}
missing = if set(translations) - installed_lang
if missing:
    raise UserError(
        _("The following language is not activated: %(missing_names)s",
        missing_names=', '.join(missing)
    )
```